src/esm_tests/repos.py

```diff
@@ -10,6 +10,13 @@
 def update_resources_submodule(info, verbose=True):
     """
     Initiates and updates the module ``esm_tests_info``.
+
+    Parameters
+    ----------
+    info : dict
+        Dictionary with the general info about the tests.
```

Comment:
It may be nice to include some example of which info must be included. That can go either (or both) in the function body at the beginning of this function which throws "friendly" key errors, and/or in the doc-string.

Reply:
`info` is a dictionary cotainning most of the information of the tests. It is build sequentially through the `cli.py`, starting with the loading of the `argpars` variables, and continuing with all the functions of esm_tests. It's similar to the `config` dictionary/object in ESM-Tools. Therefore, the description should be in a more general place, like in the `cli.py` right? Or maybe I just need to be build an `dict` subclass and put the docstrings there?

Reply:
A dict subclass is a nice idea here. You can have it fail during initialisation if it Durante have the proper parts.